package com.github.fantasticlab.rpc.core.exception;

public class InvokeException extends Exception {
}
